Update command-line-tools.rst (#3361)
mattermost/docs: source/administration/command-line-tools.rst

    -  `mattermost webhook list`_ - List all webhooks.

>     -  `mattermost webhook modify-incoming`_ - Modify an existing incoming webhook by changing its title, description, channel or icon url.

Reviewer: ```suggestion
    -  `mattermost webhook modify-incoming`_ - Modify an existing incoming webhook by changing its title, description, channel, or icon URL.
```

>     -  `mattermost webhook modify-outgoing`_ - Modify an existing outgoing webhook by changing its title, description, channel, icon, url, content-type, and triggers.

Reviewer: ```suggestion
    -  `mattermost webhook modify-outgoing`_ - Modify an existing outgoing webhook by changing its title, description, channel, icon, URL, content-type, and triggers.
```

>     -  `mattermost webhook move-outgoing`_ - Move an existing outgoing webhook with an id.

Reviewer: ```suggestion
    -  `mattermost webhook move-outgoing`_ - Move an existing outgoing webhook with an ID.
```

>           --icon [iconURL]              Icon URL 

Reviewer: ```suggestion
         --icon [iconURL]              Icon URL 
```

Reviewer: Need to confirm whether this suggestion does bring the commands into alignment.

	  --url [callbackURL]           Callback URL 
	  --content-type string         Content type 
	  
mattermost webhook move-outgoing
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  Description

>     Move an existing outgoing webhook to another team and channel by by specifying its id. If the webhook to be moved has a channel id, it must have a new one specified when moving to a new team. 

Reviewer: ```suggestion
    Move an existing outgoing webhook to another team and channel by specifying its ID. If the webhook you're moving already has a channel ID, a new one must be specified when moving it to the new team. 
```

Reviewer: Question: I may not understand the context but the example below doesn't look like it includes a new ID when moving it to a new channel. Also, is it possible for a webhook not to have an ID (i.e., "if it already has an ID") or are we saying that a webhook has an ID but can also have a channel ID?

Author: @justinegeffen I rewrote the description, does that clarify the requirements on the channel ID for you?  Also updated the example to be more explicit on the webhook ID and channel id/name.  Let me know if this is better?

Reviewer: That's awesome, thank you. :)
